### PR TITLE
GG-38603 Backport SQL System Views from Ignite

### PR DESCRIPTION
**Pull request by ygerzhedovich**

### CODE REVIEW

File: modules/core/src/main/java/org/apache/ignite/internal/managers/systemview/walker/AtomicReferenceViewWalker.java [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2019 GridGain Systems, Inc. and Contributors.
```
Comment (ygerzhedovich) [inline modules/core/src/main/java/org/apache/ignite/internal/managers/systemview/walker/AtomicReferenceViewWalker.java:2]: what reason? here and below

File: modules/core/src/main/java/org/apache/ignite/internal/managers/systemview/walker/SqlQueryHistoryViewWalker.java [hunk 2]
```diff
@@ -37,15 +37,15 @@ public class SqlQueryHistoryViewWalker implements SystemViewRowAttributeWalker<S
         v.accept(5, "durationMin", long.class);
         v.accept(6, "durationMax", long.class);
         v.accept(7, "lastStartTime", Date.class);
-        v.accept(8, "diskAllocationMax", long.class);
```
Comment (ygerzhedovich) [inline modules/core/src/main/java/org/apache/ignite/internal/managers/systemview/walker/SqlQueryHistoryViewWalker.java:40]: why did you reorder them?

**Review by ygerzhedovich — COMMENTED** [review]
The ticket not only about BINARY_METADATA, but also for TABLE_COLUMNS, VIEW_COLUMNS and VIEWS views.

File: modules/core/src/main/java/org/apache/ignite/internal/managers/systemview/walker/AtomicReferenceViewWalker.java (hunk 1 above)
Comment (zstan) [reply modules/core/src/main/java/org/apache/ignite/internal/managers/systemview/walker/AtomicReferenceViewWalker.java]: it`s auto generated code, ok i revert it.

**Comment by zstan** [comment]
@ygerzhedovich all other views are implemented, check H2IndexesSystemViewTest#testGeometryColumnPrecisionAndScale or use context search for founding particular test

File: modules/core/src/main/java/org/apache/ignite/internal/managers/systemview/walker/SqlQueryHistoryViewWalker.java (hunk 2 above)
Comment (zstan) [reply modules/core/src/main/java/org/apache/ignite/internal/managers/systemview/walker/SqlQueryHistoryViewWalker.java]: missed, thanks fixed

**Pull request merged**